cocos/2d/components/ui-opacity.ts

```diff
@@ -84,6 +84,10 @@ export class UIOpacity extends Component {
 
     // for UIOpacity
     public static setEntityLocalOpacityDirtyRecursively (node: Node, dirty: boolean, interruptParentOpacity: number) {
+        if (!node) {
```

Comment:
why need this ?

Reply:
![image](https://user-images.githubusercontent.com/32831993/186632873-cd869296-116d-4887-ae73-0c817a1349b0.png)
![image](https://user-images.githubusercontent.com/32831993/186632925-9f5117c8-0b12-4cd8-92fd-98155b43ef94.png)